src/api/wayfire/view_access_interface.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef VIEW_ACCESS_INTERFACE_HPP
```

Comment:
use `#pragma once`, that's what we use for everything new.

Reply:
Again, would have been nice to have been told this up front. Maybe doc the style guide in the wiki?  Just a bit... :)